TgeeT/Questioner_v2
# Ft admin create meetup 163283771

#### What does this PR do?

This PR create an endpoint for an admin user to create meetups
#### Description of Task to be completed?

- Ensure an  admin user should be able to create meetups

#### How should this be manually tested?
How should this be manually tested?

follow the steps below:
1. git clone this repository [here](https://github.com/TgeeT/Questioner_v2.git) 
2. Create a virtual environment and activate it.
3. git checkout to ft-admin-create-meetup-163283771 branch
4. Install requirements with 'pip install -r requirements.txt'
5. Run the application and with post man, post at the endpoint 'api/v2/meetup'

### What are the relevant pivotal tracker stories?

https://www.pivotaltracker.com/story/show/163283771

### Screenshots
###### postman
![image](https://user-images.githubusercontent.com/45405036/51540671-1472a300-1e68-11e9-94d3-5267ce081302.png)
